### DISCUSSION TITLE
Add line numbers to code blocks

### DISCUSSION THREAD

**Discussion by Isaac-Tait** [remarkjs/remark#642] [Ideas] [answered]
I am interested in enabling line numbers within code blocks. Possibly similar to [hackmd.io](https://hackmd.io) (where you can append an '=' sign after the code block intro ``` and then line numbers are rendered). 

Perhaps this is already done or possible with another remark plugin? If it is I would rather use that solution. If it is not possible yet, then I would like to build a remark plugin to enable this functionality...

**Comment by Isaac-Tait** [comment]
For reference [this](https://github.com/svelte-society/sveltesociety.dev/issues/33) issue at Svelte Society. According to the mdsvex docs the engine utilizes [remark plugins](https://mdsvex.com/docs#remarkplugins--rehypeplugins)

**Comment by ChristianMurphy** [comment, marked as answer]
[remark-prism](https://github.com/sergioramos/remark-prism) supports [prism plugins](https://prismjs.com/plugins/) including the line [number plugin](https://prismjs.com/plugins/line-numbers/).
Though you may not need to add the remark plugin, as you appear to be using [mdsvex, which has prism built in already](https://mdsvex.com/docs#highlight).

**Reply by Isaac-Tait** [reply]
Thank you @ChristianMurphy I was not aware of Prism having that functionality 😀 It is very helpful information.
